rclcpp/src/rclcpp/callback_group.cpp

```diff
@@ -97,6 +95,22 @@ CallbackGroup::automatically_add_to_executor_with_node() const
   return automatically_add_to_executor_with_node_;
 }
 
+std::shared_ptr<rclcpp::GuardCondition>
+CallbackGroup::create_notify_guard_condition(
+  const rclcpp::Context::SharedPtr context_ptr)
+{
+  notify_guard_condition_ =
+    std::make_unique<rclcpp::GuardCondition>(context_ptr);
```

Comment:
It is technically ok as-is, but I'm curious why you used `make_unique` instead of `make_shared`?

Reply:
Sorry. I used the `unique_ptr` for the member at the beginning. For some reason, I changed to `shared_ptr` for it but forgot to update this code.